src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -811,12 +811,12 @@ protected DynamoModel(IStartConfiguration config)
                         if(ext is IExtensionSource)
                         {
                            foreach(var loadedExtension in((ext as IExtensionSource).RequestedExtensions))
-                            {
-                                if(loadedExtension is IExtension)
-                                {
-                                    (loadedExtension as IExtension).Startup(startupParams);
-                                }
-                            }
+                           {
+                               if(loadedExtension is IExtension)
+                               {
+                                   (loadedExtension as IExtension).Startup(startupParams);
+                               }
+                           }
```

Comment:
Since you are at it, maybe you could fix the spacing between keywords and parentheses and the additional parentheses `in((ext as IExtensionSource).RequestedExtensions)` .